pom.xml

```diff
@@ -274,42 +272,21 @@
         <version>2.5.3</version>
       </plugin>
 
-      <!-- IMPORTANT: This plugin should always appear AFTER maven-release-plugin in this pom.xml -->
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-antrun-plugin</artifactId>
-        <version>3.0.0</version>
-        <executions>
-          <!-- Copy javadoc's "element-list" file to "package-list" in order to allow projects    -->
-          <!-- using versions of the javadoc tool < JDK9 to properly link to Thymeleaf's javadoc -->
-          <!-- as an external link.                                                               -->
-          <!-- NOTE this modifies the javadoc AFTER the creation of the jar as it is not possible -->
-          <!-- to do so by previously configuring javadoc:javadoc in the prepare-package phase    -->
-          <!-- because this "package-list" modification would trigger a second execution of the   -->
-          <!-- javadoc tool in the javadoc:jar goal.                                              -->
-          <execution>
-            <id>copy-javadoc-element-list-to-package-list</id>
-            <phase>package</phase>
-            <goals>
-              <goal>run</goal>
-            </goals>
-            <configuration>
-              <target name="copy javadocs element-list to package-list">
-                <copy file="${project.build.directory}/apidocs/element-list" tofile="${project.build.directory}/apidocs/package-list" failonerror="false" />
-              </target>
-            </configuration>
-          </execution>
-        </executions>
-      </plugin>
-
-
```

Comment:
This antrun task is the fix for #741. Why was it removed?

Reply:
Sorry this was caused by me using the wrong JDK to build. I have reverted this.